packages/cashier/src/stores/general-store.ts

```diff
@@ -318,19 +313,6 @@ export default class GeneralStore extends BaseStore {
         this.setIsP2pVisible(!(is_p2p_restricted || this.root_store.client.is_virtual));
     }
 
-    setP2pCompletedOrders(p2p_completed_orders: P2PAdvertInfo[]): void {
-        this.p2p_completed_orders = p2p_completed_orders;
-    }
-
-    async getP2pCompletedOrders() {
-        await this.WS.authorized.send?.({ p2p_order_list: 1, active: 0 }).then(response => {
-            if (!response?.error) {
-                const { p2p_order_list } = response;
-                this.setP2pCompletedOrders(p2p_order_list?.list || []);
-            }
-        });
-    }
```

Comment:
Moved the method out of `cashier` as it's not related to the cashier and has only been used in `NotificationStore`.